app/javascript/shared/components/tags.jsx

```diff
@@ -488,14 +488,18 @@ export class Tags extends Component {
   }
 }
 
+Tags.defaultProps = {
+  listing: '',
+  category: '',
+};
```

Comment:
I would add an empty line here

```suggestion
};

````